src/app/portlets/dot-edit-page/layout-properties/dot-layout-properties-item/dot-layout-properties-item.component.scss

```diff
@@ -0,0 +1,34 @@
+@import "variables";
+
+:host { 
+    display: block;
+    cursor: pointer;
+    line-height: 28px;
+    display: flex;
+    padding: $basic-padding $basic-padding * 2;
+
+    &:hover {
+        background-color: $gray-lighter; 
+    }
+}
+
+.property-item-icon {
+    width: 24px;
+    height: 24px;
+
+    .property-item-icon__iconGroup {
```

Comment:
Remember that BEM classes don't need to be nested.

Also... I think for **this case** we don't need the class inside the ```<svg>``` I see the svg as the icon and the icon as one element, so instead of adding this class you can do like:

```
.property-item-icon {
    & > g {
        // whatever.
    }
}
```

Thoughts @luisrojascr, @freddyucv and @hmoreras